modules/ROOT/pages/rest-sdk/configure-operations.adoc

```diff
@@ -5,16 +5,15 @@ endif::[]
 
 == Operation Identifiers
 
-Use an operation identifier in REST SDK to generate classes for resources.
-Fetch an operation identifier from either an OAS model
-under `operationId` or a RAML model. If fetched from a RAML model, the operation
-identifier defaults to null.
+REST SDK relies on an operation identifier to generate classes for resources.
+Fetch an operation identifier from an OAS model under `operationId`. If fetched
+from a RAML model, the operation identifier defaults to null.
```

Comment:
Hi again, this is not correct.
The `operationId` from OAS might come with a value, whereas from a RAML it will come as `null`, but the *operation identifier* we use for the resources will never be null.
The REST SDK either uses the `operationId` if it comes with a value, or defaults to the HTTP METHOD+resource URI.